packages/console/src/components/Callback/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { useLogto } from '@logto/react';
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const Callback = () => {
+  const { isAuthenticated, isLoading } = useLogto();
+  const navigate = useNavigate();
+
+  // TO-DO: Error handling
+  useEffect(() => {
+    if (isAuthenticated && !isLoading) {
+      navigate('/', { replace: true });
+    }
+  }, [isAuthenticated, isLoading, navigate]);
+
+  return <p>Redirecting...</p>;
+};
+
+export default Callback;
```

Comment:
nit: Shouldn't this be in the `pages` folder?

Reply:
yes